lib/src/scss-shared/_nx-table-variables.scss

```diff
@@ -9,5 +9,5 @@
 // nx-tables css
 $table-cell-vertical-padding: var(--nx-spacing-3x);
 $table-cell-header-height: 48px;
-$table-cell-min-height: 48px; //content box only
+$table-cell-min-height: calc(var(--nx-font-size) * var(--nx-line-height) + #{$table-cell-vertical-padding} * 2 + 1px);
```

Comment:
Just to confirm here, the `table-cell-min-height` is moving from 48px to 49px, because it will now include the 1px border bottom?

Reply:
> Just to confirm here, the table-cell-min-height is moving from 48px to 49px, because it will now include the 1px border bottom?

Yes